up42/asset.py

```diff
@@ -30,35 +30,30 @@ class Asset:
     ```
     """
 
-    def __init__(self, auth: Auth, asset_id: str, asset_info: Optional[dict] = None):
+    def __init__(
+        self,
+        auth: Auth,
+        asset_id: Optional[str] = None,
```

Comment:
str | None

Reply:
> str | None

this is not supported in Python 3.8
SDK has to support python 3.8